src/app/app.component.spec.ts

```diff
@@ -29,7 +29,7 @@ describe('AppComponent', () => {
   let el: HTMLElement;
   let store: Store<any>;
   const userinfo = new Userinfo();
-  userinfo.name = 'Joey JoJo Jr Shabadoo';
+  userinfo['name'] = userinfo['email'] = 'Joey@Shabadoo.pizza';
```

Comment:
(kept `userinfo.name` in this test so it would pass while I'm still waiting on styleguide update to be published)